components/alert/alert.html

```diff
@@ -0,0 +1,130 @@
+            </div>
+            <div class="alert-warning alert">
+                <p>Warning alert message!!!</p>
+            </div>
+            <div class="alert-danger alert">
+                <p>Danger alert message!!!</p>
+            </div>
+            <div class="alert-info alert">
+                <p>Info alert message!!!</p>
+            </div>
+            <div class="alert-primary alert">
+                <p>Primary alert message!!!</p>
+            </div>
+            <div class="alert-secondary alert">
+                <p>Secondary alert message!!!</p>
+            </div>
+            <br>
+            <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28233%2C233%2C244%2C1%29&t=cobalt&wt=none&l=django&width=700&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522alert-success%2520alert%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%253ESuccess%2520alert%2520message%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%250A%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522alert-warning%2520alert%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%253EWarning%2520alert%2520message%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%250A%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522alert-danger%2520alert%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%253EDanger%2520alert%2520message%2520%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%250A%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522alert-info%2520alert%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%253EInfo%2520alert%2520message%253C%252Fp%253E%250A%2509%2520%2520%253C%252Fdiv%253E%250A%250A%2509%2520%2520%253Cdiv%2520class%253D%2522alert-primary%2520alert%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%253EPrimary%2520alert%2520message%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%250A%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522alert-secondary%2520alert%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%253ESecondary%2520alert%2520message%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E"
+            style="width: 700px; height: 597px; border:0; transform: scale(1); overflow:hidden;"
+            sandbox="allow-scripts allow-same-origin">
+          </iframe>
+        </div>
+
+    </div>
+
+
+
+    <!---------------------------------main context end------------------->
+
+
+    <!--------------------------------footer starts------------------------>
+    <footer class="footer">
+
+        <h2>Made with ❤️ by shubham</p>
+            <h3>connect with me 👇 </p>
+                <a href="https://github.com/Shubhamkr2610">github<i class="fab fa-github-square"></i></a>
```

Comment:
Always use target=_blank with the anchor attributes for external links

Reply:
I implemented this to my code. please check and review it